core/src/test/java/de/danoeh/antennapod/core/storage/ItemEnqueuePositionCalculatorTest.java

```diff
@@ -249,7 +249,8 @@ static void doAddToQueueAndAssertResult(String message,
         assertEquals(message, idsExpected, getIdList(queue));
     }
 
-    static final List<FeedItem> QUEUE_EMPTY = Collections.unmodifiableList(Arrays.asList());
+    static final List<FeedItem> QUEUE_EMPTY = Collections.unmodifiableList(
+            Collections.emptyList());
```

Comment:
`Collections.emptyList` is already unmodifiable. There is no need to wrap it again.